lib/php/metadata-manipulator-bundle/tests/MetadataManipulatorTest.php

```diff
@@ -15,28 +14,25 @@ class MetadataManipulatorTest extends TestCase
     /**
      * @covers MetadataManipulator::getKnownTagGroups
      */
-    public function testGetKnownTagGroups(): void
-    {
-        self::assertIsArray($this->service->getKnownTagGroups());
-    }
+//    public function testGetKnownTagGroups(): void
+//    {
+//        self::assertIsArray($this->service->getKnownTagGroups());
+//    }
```

Comment:
remove?

Reply:
getKnownTagGroups is to be re-implemented later (to populate admin ux ; select with list of tag).
I will change test to "incomplete"